bach/tests/functional/bach/test_series_numeric.py

```diff
@@ -357,3 +357,27 @@ def test_series_minmax_scale() -> None:
         expected_data=expected_data_default,
         round_decimals=True,
     )
+
+
+def test_exp() -> None:
+    skating_order = get_bt_with_test_data(full_data_set=True)['skating_order']
```

Comment:
```suggestion
def test_exp(engine) -> None:
    skating_order = get_df_with_test_data(engine, full_data_set=True)['skating_order']
```

`get_bt_with_test_data` is deprecated as it only works with Postgres, and we want to run all tests on both Postgres and BigQuery